### PR TITLE
[4.0] TinyMCE context menu

### PR DESCRIPTION
**Pull request by brianteeman**
Tinymce has an option in the plugin called Context Menu but it doesn't do anything as there is no code to do anything.

With this PR you can no disable/enable the ability to right click on an item in the editor and get a contextual menu. When this is disabled you get the context menu from your browser
#### Enabled
image

#### Disabled
image

### Testing
Make sure that you set the option to be enabled/disabled in the correct set. Default for a super user is set 0

### CODE REVIEW

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 0ca3a103e1c61a41a53cf2ea1fc92c28f2a8ac02<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27497">issues.joomla.org/tracker/joomla-cms/27497</a>.</sub>

File: plugins/editors/tinymce/tinymce.php [hunk 1]
```diff
@@ -598,6 +598,7 @@ public function onDisplay(
 				'templates'          => $templates,
 				'image_advtab'       => (bool) $levelParams->get('image_advtab', false),
 				'external_plugins'   => empty($externalPlugins) ? null  : $externalPlugins,
+				'contextmenu'        => (bool) $levelParams->get('contextmenu', true) ? null : false,
```
Comment (Fedik) [inline plugins/editors/tinymce/tinymce.php:601]: it enough just `'contextmenu'  => (bool) $levelParams->get('contextmenu', true),`

Comment (brianteeman) [reply plugins/editors/tinymce/tinymce.php:601]: no that does not work

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on 0ca3a103e1c61a41a53cf2ea1fc92c28f2a8ac02<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27497">issues.joomla.org/tracker/joomla-cms/27497</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27497">issues.joomla.org/tracker/joomla-cms/27497</a>.</sub>

**Comment by HLeithner** [comment]
Thanks

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**